scripts/create_spec_repo/firebase_sdk.textproto

```diff
@@ -0,0 +1,9 @@
+# Config file to update versions of Pods which will be pushed to SpecsStaging
+# repo by add_podspec tool.
+# e.g.
+# Podspec versions will be the same as firebase-ios-sdk podspecs if they are not
+# specified in this file.
+sdk {
+  sdk_name: "Firebase"
+  sdk_version: "6.31.0"
```

Comment:
Why is the Firebase version specified here versus the one from the Firebase.podspec in the repo?

Reply:
This is due to a potential nil.
https://github.com/firebase/firebase-ios-sdk/blob/master/ZipBuilder/Sources/firebase-pod-updater/main.swift#L52
Has update the pod-version-updator to allow textproto to be empty.

Reply:
275fddf